addons/languages/songhay/apk/build.gradle

```diff
@@ -0,0 +1,7 @@
+ext.shouldBePublished = false
```

Comment:
Please also add another line for the reason of not publishing:
```
ext.notPublishingReason = "Initial version has not been published yet"
```

Reply:
Thank you. The line has been added.